Formula/ssh-agent-filter.rb

```diff
@@ -0,0 +1,62 @@
+class SshAgentFilter < Formula
+  desc "filtering proxy for ssh-agent"
+  homepage "https://github.com/tiwe-de/ssh-agent-filter"
+  url "https://github.com/tiwe-de/ssh-agent-filter/archive/0.4.2.tar.gz"
+  sha256 "5586e40da34f4afd873d7ff30f115c4a1b9455c1b81f3c4679c995db7615b080"
+
+  depends_on "help2man" => :build
+  depends_on "boost"
+  depends_on "nettle"
+
+  patch :DATA
+
+  def install
+    ENV.deparallelize
+    system "make", "ssh-agent-filter.1"
+    man1.install "ssh-agent-filter.1"
```

Comment:
It seems like you only want to install some of the files the Debian package contains. The missing parts are at least:
- afssh.1 (needs pandoc to generate it)
- ssh-agent-filter.bash-completion (which version of bash-completion is available on OS X?)